src/env/test/mod.rs

```diff
@@ -113,4 +117,8 @@ impl Env for TestEnv {
     fn write(&mut self) -> Self::Write {
         TestWrite
     }
+
+    fn customization(&self) -> &Customization {
```

Comment:
This will need to be `mut`.

Reply:
I exposed `customization_mut` in `impl TestEnv` now instead of in trait impl.